CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] Adds bearer token support for mimirtool's analyze ruler/prometheus commands.
```

Comment:
Could you move this down to group it with the other `ENHANCEMENT`s?

Reply:
Done!